VaticanBAV/et/BAVet126.xml

```diff
@@ -7,7 +7,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <titleStmt>
                 <title>Catholic Epistles</title>
                 <editor role="generalEditor" key="AB"/>
-                <editor key="DE"/>
+                <editor key="MKr"/>
```

Comment:
If you mean Magda she is MK

Reply:
I mean Marcin

Reply:
MKr is Marcin and is correct

Reply:
list of abbreviation is visible from the team page https://betamasaheft.eu/team.html